Dockerfile

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.18.3@sha256:a452d6273ad03a47c2f29b898d6bb57630e77baf839651ef77d03e4e049c5bf3 AS builder
```

Comment:
You'll need to sync from HEAD as this commit was already added to main.

Reply:
> > For PRs, it's helpful to write an informative title, like "Grammatical updates to README" and a description of the change.
> 
> I am really sorry this time but next time i do 🥺🥺❤️



> You'll need to sync from HEAD as this commit was already added to main.
I Messed up a bit so i forgot to write a description